dotnet/src/Planners/Planners.OpenAI/Stepwise/GeneratePlan.yaml

```diff
@@ -0,0 +1,39 @@
+template_format: semantic-kernel
+template: |
+  <message role=""system"">
+  You are an expert at generating plans from a given GOAL. Think step by step and determine a plan to satisfy the specified GOAL using only the FUNCTIONS provided to you. You can also make use of your own knowledge while forming an answer but you must not use functions that are not provided. Once you have come to a final answer, use the UserInteraction_SendFinalAnswer function to communicate this back to the user.
+
+  [FUNCTIONS]
+
+  {{$available_functions}}
+
+  [END FUNCTIONS]
+
+  To create the plan, follow these steps:
+  0. Each step should be something that is capable of being done by the list of available functions.
+  1. Steps can use output from one or more previous steps as input, if appropriate.
+  2. The plan should be as short as possible.
+  </message>
+  <message role=""user"">{{$goal}}</message>
+description:     Generate a step-by-step plan to satisfy a given goal
+name:            GeneratePlan
+input_parameters:
+  - name:          available_functions
+    description:   A list of functions that can be used to generate the plan
+  - name:          goal
+    description:   The goal to satisfy
+execution_settings:
+  - model_id:          gpt-4
+    temperature:       1.0
+    top_p:             0.0
+    presence_penalty:  0.0
+    frequency_penalty: 0.0
+    max_tokens:        256
+    stop_sequences:    []
+  - model_id:          gpt-3.5
+    temperature:       0.0
```

Comment:
gpt-3.5 has temp of 0.0 and gpt-4 has temp of 1.0? Have we tested with both?

Reply:
Oh, I forgot to update this. They should both be zero for best planner behavior.

Reply:
And the model IDs probably should be changed to actual ones that support function calling, instead of generic gpt3.5 / gpt4